shell/platform/fuchsia/flutter/engine.cc

```diff
@@ -354,8 +354,6 @@ void Engine::Initialize(
        view_protocols = std::move(view_protocols),
        request = parent_viewport_watcher.NewRequest(),
        view_ref_pair = std::move(view_ref_pair),
-       max_frames_in_flight = product_config.get_max_frames_in_flight(),
```

Comment:
Nice, I think this means we can get rid of the product config file soon.

Please remove these fields from the flutter_runner_product_config.h

Reply:
Done.